### DISCUSSION TITLE
Adding tooltips to icons

### DISCUSSION THREAD

**Discussion by PhilC813** [jocmp/capyreader#363] [Feature Requests]
Hello! Just discovered your app after giving Feedbin a try. It's super minimalist and well designed; loving it so far, knowing that it's still the early days.

I was wondering if you could consider adding [tooltips](https://developer.android.com/develop/ui/views/components/tooltips) (long press description) to the different icons across the app. It is helpful for new users who may not yet be familiar with the icons (other than the obvious "star" and "share").

![Screenshot_20240916_023947.jpg](https://github.com/user-attachments/assets/231f80c0-acd3-41b0-bf56-28bdc4d0bf0a)

Thank you!

**Comment by jocmp** [comment]
hey @Philippe-Choquette - thanks for checking out the app. I'll take a look. I generally want to avoid popups since they can disrupt the UI, but I hear you about the unfamiliar icons and I've also seen how Google Photos for instance uses popups on long press.

**Reply by PhilC813** [reply]
Thank you for considering this! I've also just noticed that the Feedbin website uses tooltips too; you can probably reuse the same text with the icons if you ever decide to implement.

![Screenshot_20240917_194104.jpg](https://github.com/user-attachments/assets/6392a286-875e-4260-8364-cd89a286cea8)

**Comment by jocmp** [comment]
I have a change up for this (https://github.com/jocmp/capyreader/pull/426) that I'll include in the next release. Here's a demo of what it looks like. The tooltip may appear beneath to toolbar on smaller screens.

https://github.com/user-attachments/assets/b4da71b7-4139-48a5-84a8-766828b8c0eb

**Reply by PhilC813** [reply]
That's awesome thanks! And it doesn't burden the UI!
